modules/dop/endpoints/endpoints.go

```diff
@@ -214,6 +214,7 @@ func (e *Endpoints) Routes() []httpserver.Endpoint {
 		{Path: "/api/cicd-crons/{cronID}/actions/start", Method: http.MethodPut, Handler: e.pipelineCronStart},
 		{Path: "/api/cicd-crons/{cronID}/actions/stop", Method: http.MethodPut, Handler: e.pipelineCronStop},
 		{Path: "/api/cicd-crons", Method: http.MethodPost, Handler: e.pipelineCronCreate},
+		{Path: "/api/cicd-crons/update", Method: http.MethodPost, Handler: e.pipelineCronUpdate},
```

Comment:
PUT: /api/cicd-crons/{cronID}

Reply:
eventBox call back only support post method
and the path is updated to /api/cicd-crons/actions/hook-for-update